test/e2e/default_bridged_network_test.go

```diff
@@ -142,7 +142,9 @@ func dhcpFlag(node string, name string) bool {
 func nodeReadyConditionStatus(nodeName string) (corev1.ConditionStatus, error) {
 	key := types.NamespacedName{Name: nodeName}
 	node := corev1.Node{}
-	err := framework.Global.Client.Get(context.TODO(), key, &node)
+	oneSecondTimeoutCtx, cancel := context.WithTimeout(context.Background(), 1*time.Second)
```

Comment:
This deserves an inline comment. We may need it elsewhere later, to we better document whats and whys.